.gitignore

```diff
@@ -5,7 +5,7 @@ hs_err_pid*
 
 # Gradle
 **/.gradle
-/build
+build/
```

Comment:
Please don't do this. There are several Java packages with the name `build` across Spoofax. This will ignore all of them, which will cause confusion and errors when changes in those packages are not committed.